.pipelines/upgrade-job/upgrade-templates.yaml

```diff
@@ -2,11 +2,11 @@ steps:
   - template: ../templates/get-latest-tag.yaml
   - template: ../templates/add-arc-extension.yaml
     parameters: 
-      imageTag: $(LATEST_PUBLISHED_TAG)
+      imageTag: 1.2.3 # Temporarily hard-coding
```

Comment:
question, why do we have to hardcode the version?

Reply:
There's a slight issue with the order in which the tags need to be generated now, so this is a temporary work around. I'll follow up to fix the ordering if needed.

Reply:
sounds good, could you make a task on the sprint board to track this?

Reply:
Discussed offline, but will do if needed for an upcoming release.